lessons/Early-Math/absolute-value-and-opposites/langs/en-us/index.adoc

```diff
@@ -0,0 +1,150 @@
+--
+
+When we want to write a mathematical expression that indicates that negating is happen, we just use a negative sign! So, while we can read @math{-45} by simply saying "negative 45," another way to read that value's name is by saying, "the opposite of 45."
+
+
+[.strategy-box, cols="1", grid="none", stripes="none"]
+|===
+|
+@span{.title}{Pedagogy Note: Can we call it "minus 45"?!}
+Precise language is crucial when dealing with this symbol: `-`.
+
+In math we want students to read for _comprehension_, not just mechanically sounding out each symbol in order. We read the expression @math{12 - 9} as "12 minus 9" __because subtraction is the "verb"__, not because the minus sign happens to be in the middle!
+
+When we encounter a value such as @math{-20}, however, it is confusing and misleading to name that value "minus 5." Subtraction is not happening here - negation is! But if the teacher uses the "symbols in order" wording, it reinforces the lower level of reading comprehension for students.
+|===
+
+
+
+=== Synthesize
+
+- Two opposites are 8 units apart from each other. What are they?
+** @math{4} and @math{-4}
+- Two opposites are 42.5 units apart from each other. What are they?
+** @math{21.5} and @math{-21.5}
+- Cia says that opposites are always negative. Are they right? Why?
+** _No, opposites are not always negative. When we negate a negative value, we get a positive value. Additionally, zero is its own opposite, and it is neither positive nor negative._
+
+== Absolute Value @duration{30 minutes}
+
+=== Overview
+
+Students consider the meaning of absolute value, and apply the concept to Circles of Evaluation using `num-abs`.
+
+=== Launch
+
+[.lesson-instruction]
+What is the distance between these two points on the number line: @math{-18} and @math{5}?
+
+Give students a minute to contemplate, and then invite them to verbally share their strategies. Record students' thinking the board, annotating the number line. All strategies are welcome, with a special interest in any discussion that hones in on the idea of _the distance of a number from zero._
+
+Explain to students that we have a term for _the distance of a number from zero_ - it's @vocab{absolute value}.
+
+We annotate absolute value like this: @math{|x|}, with @math{x} being any given number. When we encounter an expression like @math{|x|}, we say "the absolute value of @math{x}."
+
+Because _opposites_ are the same distance away from zero, they will always have the same absolute value. So, @math{|4| = 4} and @math{|-4| = 4}.
+
+
+=== Investigate
+
+The function to represent absolute value is `num-abs`. It can be used like any other operator that we put at the top of a Circle of Evaluation - with one important difference: when we use `negate`, we include just one argument inside the Circle.
+
+[.lesson-instruction]
+- Compare expressions with `num-abs` to expressions with `negate` on @printable-exercise{true-or-false-abs-val.adoc}.
```

Comment:
Probably makes sense to have students explore `num-abs` before comparing to `negate`

Still not 100% sure it's worth using these named functions if you're not going to have hands on keys.